src/AppBundle/Controller/UserController.php

```diff
@@ -73,4 +74,21 @@ public function avatarAction($username, Request $request)
 
         return $response;
     }
+
+    /**
+     * @Route("/register/resend-email", name="register_resend_email", methods={"POST"})
+     * @Template()
+     */
+    public function resendRegistrationEmailAction(Request $request, UserManagerInterface $userManager, MailerInterface $mailer)
+    {
+        if($request->request->get("email") !== NULL) {
+            $email = $request->request->get("email");
+            $user = $userManager->findUserByEmail($email);
```

Comment:
Problem still there: you are not checking if `$user` exists. `$user` may be NULL if the email does not exist in database.